### DISCUSSION TITLE
Silent authentication???

### DISCUSSION THREAD

**Discussion by bertoluccimauricio** [trakt/trakt-api#456] [Q&A] [answered]
I think the current authentication process (opens a screen in the browser with a Trakt login, generates a pin, enters a pin in the app) is a process that can be strange/weird in the eyes of the new user. Is there any other way to do this more naturally? In other words, like any other app, with its own login and so on...

**Comment by rectifyer** [comment, marked as answer]
What we have is pretty standard and the most secure for the user. We don't want to have app transmitting emails and passwords directly.
